### PR TITLE
Added prettier as pre-commit hook and cleaned up repo.

### PR DESCRIPTION
**Pull request by ankush38u**
Preparing for a larger release.
In this PR
 - [x] Added prettier as a pre commit hook
 - [x] Added eslint to test code for later use
 - [x] Cleaned up existing blocks with prettier, so next PR can be merged if no style errors.
 - [x] Removed 2 blocks for not following guidelines. Github checkout and UploadApkAndroid.

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Ignore artifacts:
```
Comment (shubhamsoni28) [inline .prettierignore:1]: ignore nested node_modules, dist, .dcoder and .upm directories as well

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "env": {
```
Comment (shubhamsoni28) [inline .eslintrc.json:2]: Do we need eslint if we are just doing the prettier formatting?

**Review by shubhamsoni28 — CHANGES_REQUESTED** [review]

File: .eslintrc.json (hunk 2 above)
Comment (ankush38u) [reply .eslintrc.json:2]: Haan, kept it for future usage, it is not enforced yet, but people can see which rules we are following, new blocks used with vs code started showing lint errors. Want to keep it as a good practice.

File: .prettierignore (hunk 1 above)
Comment (ankush38u) [reply .prettierignore:1]: No need, tested with nested codes in dist and node_modules in the blocks, the prettierignore rules are working on nested directories too.

**Pull request merged**